src/lib/data/bluetooth.test.ts

```diff
@@ -0,0 +1,75 @@
+import {createSensor} from '../sensor';
+import {bleSensorService} from './bluetooth';
+import {ble} from '../sensor';
+
+type Sensor = {
+  id: string;
+  name: string;
+  is_primary: boolean;
+  sensorType: string[];
+  type: string;
+  address: string;
+};
+
+beforeEach(async () => {
+  // Create our global ble object
+  try {
+    await ble.requestPermissions();
+    await ble.start();
+  } catch (err) {
+    throw new Error('problem starting ble');
+  }
+  jest.clearAllMocks();
```

Comment:
clearAllMocks is called in the global.afterEach in ./setupHooks.ts so it shouldn't be needed here.